documentation/Database-Schema-Template.md

```diff
@@ -12,11 +12,17 @@ The sections and questions required in each stage of an application is defined i
 
 Representation of the application template. All nested elements are accessible via joined tables and can be created or queried in the same call using the GraphQL engine.
 
-Whenever a template is modified, the new version is saved as a new template record with a new `version_timestamp`. Different versions of the same base template are connected with the same `code`. The `status` field is used to indicate which version is the current active one.
+Whenever a template is modified, the new version is saved as a new template record with a new `version_timestamp`. Different versions of the same base template are connected with the same `code`. The `status` field is used to indicate which version is the current available one.
+
+The `status` can be `'Draft'`, `'Available'` or `'Disabled'`. The version currently active in the system is the one (and only one) marked `'Available'`.
```

Comment:
Just a question -- have we written database logic to enforce that only one version will be "Available" like we have to application_stage_history and application_status_history? If not, perhaps create an issue to do this?